src/command_modules/azure-cli-vm/tests/test_vm_commands.py

```diff
@@ -858,12 +858,12 @@ def test_diagnostics_extension_install(self):
         self.execute()
 
     def body(self):
-        storage_key = '123'  # use junk keys, do not retrieve real keys which will get into the recording
+        storage_sastoken = '123'  # use junk keys, do not retrieve real keys which will get into the recording
         _, protected_settings = tempfile.mkstemp()
         with open(protected_settings, 'w') as outfile:
             json.dump({
                 "storageAccountName": self.storage_account,
-                "storageAccountKey": storage_key,
+                "storageAccountSasToken": storage_sastoken,
```

Comment:
Actually, no. We were pushed from the Azure Security team not to use a full storage shared access key, and storageAccountKey is now fully deprecated. That's one of the reasons why LAD 3.0 can't be backward-compatible with LAD 2.3.